src/Lunr/Vortex/FCM/FCMApnsPriority.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * This file contains priority types for Firebase Cloud Messaging Notifications to APNS.
+ *
+ * SPDX-FileCopyrightText: Copyright 2024 Move Agency Group B.V., Zwolle, The Netherlands
+ * SPDX-License-Identifier: MIT
+ */
+
+namespace Lunr\Vortex\FCM;
+
+/**
+ * Firebase Cloud Messaging APNS Priority Types.
+ */
+enum FCMApnsPriority: int
+{
+
+    /**
+     * Prioritize the device’s power considerations over all other factors for delivery,
+     * and prevent awakening the device
+     */
+    case low = 1;
+
+    /**
+     * Send the notification based on power considerations on the user’s device
+     */
+    case default = 5;
+
+    /**
+     * Send the notification immediately
+     */
+    case high = 10;
```

Comment:
enum cases should be PascalCase

Reply:
No stylecheck failed 😄.

I will make https://github.com/lunr-php/lunr.vortex/pull/12/files#diff-f930c4fe97c42bf5fc90693166070d3cdc690cfc79ea1b1bbc5891ace89b6f41 more complicated, but I guess that's fine.